codeChunks.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Frequently Used Code Chunks"
+author: "Vivian Tran"
+date: "3/7/2018"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+These are some code chunks that I frequently come back to when processing data for the Arctic Data Center. 
+
+#Reading in raw data
```

Comment:
GitHub is finicky about spaces after # for the headers, so make sure to include them!  RStudio will preview it just find, but GitHub won't.  (`#Reading`--> `# Reading`)